Dockerfile

```diff
@@ -2,17 +2,16 @@
 ARG PYTHON_VERSION="3.10.7"
 
 
-
-##############
-## Frontend ##
-##############
+###################
+## Frontend Base ##
+###################
 FROM node:16 as frontend
 
-# Build the admin-io frontend
+# Build the admin-ui frontend
 WORKDIR /fides/clients/admin-ui
-COPY clients/admin-ui/ .
+COPY clients/admin-ui/package.json clients/admin-ui/package-lock.json ./
 RUN npm install
-RUN npm run export
```

Comment:
This is moved to the "production" stage, since in development mode we always use `npm run dev` anyways (so the export is unnecessary and unused)